### PR TITLE
Add several new tools

### PR DESCRIPTION
**Pull request by ajhall**
Add several tools that I use regularly:

- age
- chezmoi
- choose
- direnv
- doggo
- gh
- krew
- kubectx
- mise
- stern
- tlrc

### CODE REVIEW

File: data/apps.csv [hunk 1]
```diff
@@ -1490,3 +1490,14 @@ prompt,promptless,,https://github.com/dylanaraps/promptless,"A suckless and supe
...
+security,age,https://age-encryption.org/,https://github.com/FiloSottile/age,"A simple, modern and secure encryption tool with small explicit keys, no config options, and UNIX-style composability."
+system,chezmoi,https://www.chezmoi.io/,https://github.com/twpayne/chezmoi,"Manage your dotfiles across multiple diverse machines, securely."
+text-processing,choose,,https://github.com/theryangeary/choose,A human-friendly and fast alternative to cut and (sometimes) awk.
+system,direnv,https://direnv.net/,https://github.com/direnv/direnv,Loads and unloads environment variables depending on the current directory.
+networking,doggo,https://doggo.mrkaran.dev/,https://github.com/mr-karan/doggo,Command-line DNS client for humans.
+package-manager,krew,https://krew.sigs.k8s.io/,https://github.com/kubernetes-sigs/krew,Find and install kubectl plugins.
+devops,kubectx,https://kubectx.dev/,https://github.com/ahmetb/kubectx,Faster way to switch between clusters and namespaces in kubectl.
+package-manager,mise,https://mise.jdx.dev/,https://github.com/jdx/mise,"A development environment setup tool: dev tools, env vars, and task runner."
```
Comment (vitorhcl) [inline data/apps.csv:1500]:
```suggestion
package-manager,mise,https://mise.jdx.dev/,https://github.com/jdx/mise,"A development environment setup tool: dev tools, env vars, and task runner. Like `asdf` + `direnv` + `make`"
```
Just a minor suggestion.

File: data/apps.csv [hunk 2]
```diff
@@ -1490,3 +1490,14 @@ prompt,promptless,,https://github.com/dylanaraps/promptless,"A suckless and supe
...
 transfer,osync,http://www.netpower.fr/osync,https://github.com/deajan/osync,"A robust two-way (bidirectional) file sync script based on rsync with fault tolerance, POSIX ACL support, time control and near real-time sync."
 email,mailsy,,https://github.com/BalliAsghar/Mailsy.git,Generates disposable emails in the CLI through [mail.tm](https://mail.tm).
+security,age,https://age-encryption.org/,https://github.com/FiloSottile/age,"A simple, modern and secure encryption tool with small explicit keys, no config options, and UNIX-style composability."
+system,chezmoi,https://www.chezmoi.io/,https://github.com/twpayne/chezmoi,"Manage your dotfiles across multiple diverse machines, securely."
+text-processing,choose,,https://github.com/theryangeary/choose,A human-friendly and fast alternative to cut and (sometimes) awk.
+system,direnv,https://direnv.net/,https://github.com/direnv/direnv,Loads and unloads environment variables depending on the current directory.
+networking,doggo,https://doggo.mrkaran.dev/,https://github.com/mr-karan/doggo,Command-line DNS client for humans.
```
Comment (vitorhcl) [inline data/apps.csv:1497]:
```suggestion
networking,doggo,https://doggo.mrkaran.dev/,https://github.com/mr-karan/doggo,"DNS client for humans. Features include: colors, tabular and JSON formats, and reverse DNS lookup"
```
We don't need to specify it's a command-line client, all tools of this repo are :sweat_smile:

File: data/apps.csv [hunk 3]
```diff
@@ -1490,3 +1490,14 @@ prompt,promptless,,https://github.com/dylanaraps/promptless,"A suckless and supe
...
+package-manager,krew,https://krew.sigs.k8s.io/,https://github.com/kubernetes-sigs/krew,Find and install kubectl plugins.
+devops,kubectx,https://kubectx.dev/,https://github.com/ahmetb/kubectx,Faster way to switch between clusters and namespaces in kubectl.
+package-manager,mise,https://mise.jdx.dev/,https://github.com/jdx/mise,"A development environment setup tool: dev tools, env vars, and task runner."
+devops,stern,,https://github.com/stern/stern,Multi pod and container log tailing for Kubernetes.
+git,gh,https://cli.github.com/,https://github.com/cli/cli,GitHub's official command line tool.
```
Comment (vitorhcl) [inline data/apps.csv:1502]:
```suggestion
git,gh,https://cli.github.com/,https://github.com/cli/cli,GitHub's official tool to manage repos, issues, projects, gists and much more.
```

**Comment by ajhall** [comment]
Thanks, good suggestions! I rewrote my commits with your wording and re-pushed my branch.

**Pull request merged**